src/providers/ContentShareProvider/docs/ContentShareProvider.mdx

```diff
@@ -63,7 +128,7 @@ import { ContentShareProvider } from 'amazon-chime-sdk-component-library-react';
 
 const App = () => (
   <CustomAudioVideoProvider>
-    <ContentShareProvider>
+    <ContentShareProvider maxContentShares={2}>
```

Comment:
Its fine to drop the drop from here as its covered under the example updated above.

Reply:
updated